admin/client/src/components/MessageBox/MessageBox.js

```diff
@@ -0,0 +1,105 @@
+import React, { PropTypes } from 'react';
+import SilverStripeComponent from 'lib/SilverStripeComponent';
+import { Alert } from 'react-bootstrap-ss';
+import castStringToElement from 'lib/castStringToElement';
+
+/**
+ * A wrapper for Alert messages in react-bootstrap.
+ * Displays a given message.
+ */
+class MessageBox extends SilverStripeComponent {
```

Comment:
It's no more related to forms than then underlying `<Alert>` - we just happen to add a way to persist dismissal, and translate our own message type terminology into bootstrap terminology.